cmd/minishift/cmd/start_test.go

```diff
@@ -361,11 +362,11 @@ func TestDetermineIsoUrl(t *testing.T) {
 		in  string
 		out string
 	}{
-		{"", constants.DefaultB2dIsoUrl},
-		{"b2d", constants.DefaultB2dIsoUrl},
-		{"B2D", constants.DefaultB2dIsoUrl},
-		{"centos", constants.DefaultCentOsIsoUrl},
-		{"CentOs", constants.DefaultCentOsIsoUrl},
+		{"", minishiftUtil.GetLatestISOVersion("b2d")},
+		{"b2d", minishiftUtil.GetLatestISOVersion("b2d")},
+		{"B2D", minishiftUtil.GetLatestISOVersion("b2d")},
+		{"centos", minishiftUtil.GetLatestISOVersion("centos")},
+		{"CentOs", minishiftUtil.GetLatestISOVersion("centos")},
```

Comment:
`CentOs` is `CentOS` (as OS is an abbreviation for Operating System)

Note: just a test...